src/core/ext/filters/client_channel/lb_policy/grpclb/proto/grpc/lb/v1/load_balancer.pb.h

```diff
@@ -4,6 +4,7 @@
 #ifndef PB_GRPC_LB_V1_LOAD_BALANCER_PB_H_INCLUDED
 #define PB_GRPC_LB_V1_LOAD_BALANCER_PB_H_INCLUDED
 #include "pb.h"
+#include "grpc/impl/codegen/port_platform.h"
```

Comment:
Nit: `#include <grpc/impl/codegen/port_platform.h>`

This is a nanppb-generated file though.

This file is actually being deleted soon in favor of upb. So I'm fine changing this generated file to silence some warning.